### ISSUE TITLE
not able to load SqueezeNet_v1.1.mlmodel to xcode

### ISSUE THREAD

**Issue by steve21124** [mdering/CoreMLZoo#1]
are you able to load SqueezeNet_v1.1.mlmodel to xcode? seems like xcode not recognizing the mlmodel.

**Comment by alxndrTL** [comment]
No, i also have this problem, and it's the same thing for the VGG19 model and de SqueezeNet_v1.0.mlmodel 
Error : "There was an problem decoding this CoreML document"
